docs/getting-started/README.md

```diff
@@ -40,4 +40,4 @@ var chart = new Chart(ctx, {
 
 It's that easy to get started using Chart.js! From here you can explore the many options that can help you customise your charts with scales, tooltips, labels, colors, custom actions, and much more.
 
-There are many examples of Chart.js that are available in the `/samples` folder of `Chart.js.zip` that is attached to every [release](https://github.com/chartjs/Chart.js/releases).
\ No newline at end of file
+There are many examples of Chart.js that are available in the `/samples` folder of `Chart.js.zip` that is attached to every [release](https://github.com/chartjs/Chart.js/releases).
```

Comment:
Downloading a zip seems like a cumbersome way to see the samples. What about?

There are many [examples of Chart.js that are available on our website](https://www.chartjs.org/samples/latest/) and you can see the [source code for these samples on GitHub](https://github.com/chartjs/Chart.js/tree/master/samples).

Reply:
We actually added this line to make sure users will not report the `master` samples not working. Most of them just download the GitHub sources or clone master to check samples and then open a ticket saying they don't work.

Why not this instead:

> All our examples are [available online](https://www.chartjs.org/samples/latest/) but you can also download the `Chart.js.zip` archive attached to every [release](https://github.com/chartjs/Chart.js/releases) to experiment with our samples locally from the `/samples` folder.

Reply:
Ok. I like that